cmd/mattermost/commands/team.go

```diff
@@ -247,3 +257,29 @@ func listTeamsCmdF(command *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func searchTeamCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+
+	teams := make([]*model.Team, 0, len(args))
+
+	for _, searchTerm := range args {
+		team, err := a.SearchAllTeams(searchTerm)
```

Comment:
the team variable should be something like "teams", but here, you can use something like foundTeams.